api/urls.py

```diff
@@ -5,4 +5,5 @@
 urlpatterns = [
     path("", views.BrandSuggestionAPIView.as_view()),
     path("bank-contacts/", views.ContactView.as_view(), name="contacts"),
+    path("bank/", views.BrandsView.as_view(), name="bank"),
```

Comment:
can you remove trailing slash here? django automatically appends  trailing slash.

Reply:
done, I was following the precedent of the "bank-contacts/", should I remove that one as well?

Reply:
My bad, keep trailing slash for bank url as well.